lib/subscribers/mysql/connection-add-command.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2025 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const PropagationSubscriber = require('../propagation')
+const { EventEmitter } = require('node:events')
+
+class MySQL2ConnectionAddCommandSubscriber extends PropagationSubscriber {
+  constructor({ agent, logger, channelName = 'nr_connectionAddCommand', packageName = 'mysql2' }) {
+    super({ agent, logger, packageName, channelName, callback: null })
+  }
+
+  handler(data, ctx) {
+    const newCtx = super.handler(data, ctx)
```

Comment:
this isn't needed. the super is `base.js` and its handler is a no-op. I assume you copied this from `pg` which calls the super because it makes the db segment then wraps the event emitter to propagate the context.